components/producers/dependency-check/examples/README.md

```diff
@@ -0,0 +1,12 @@
+# Examples
+
+## requirements.txt
+
+```bash
+brew update && brew install dependency-check
```

Comment:
is this mac-specific?

Reply:
no, actually you can use `brew` on linux too! But yeah, agreed - I'll tweak the wording